### ISSUE TITLE
Docs improvements

### ISSUE THREAD

**Issue by bjsi** [OpenPipe/openpipe#366]
The docs are great, setting things up is simple. These are some nitpicks:

- In what scenarios do people generally use tags? Would be good to list the top examples. Eg. most people will probably want to tag by the name/purpose of the prompt in their code.

<img width="914" alt="image" src="https://github.com/OpenPipe/OpenPipe/assets/58147075/3c0f5127-f9f4-4ecd-b65f-d953db9929df">

- These look clickable but aren't

<img width="839" alt="image" src="https://github.com/OpenPipe/OpenPipe/assets/58147075/4799b1ff-943b-44ff-8d90-af8a7738cf49">

- Maybe: Add pics here? Especially for swapping OpenAI models with the OpenPipe fine tune because that shows how easy the migration is.

<img width="627" alt="image" src="https://github.com/OpenPipe/OpenPipe/assets/58147075/6cd1b444-e06c-44a4-9b1c-11149076fa16">

**Comment by arcticfly** [comment]
Hey @bjsi, thanks for pointing these out! I'll take care of the non-clickable boxes this evening, and add some more color on why searchable tags are useful.

I'm not sure about adding pics to the features list. Large pictures would make it harder to get through the list, and small pictures might be too difficult to understand. I'm open to other opinions.

**Comment by arcticfly** [comment]
Okay, I added a little to the search tags explanation, lmk what you think. I also made those boxes clickable (really good idea!). Now, on the subject of adding pictures to the overview page - I think adding links to the actual feature pages within the docs is a good middle ground. It's easy to see an overview of all the features, but if you want to visualize how a feature works, you just have to click on the title and you'll see a screenshot on the feature's page. Wdyt?
